pkg/config/repoconfig/repoconfig_suite_test.go

```diff
@@ -0,0 +1,89 @@
+package repoconfig
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("repoconfig parsing", func() {
+	When("the yaml file has no label-approved", func() {
+		It("should return no LabelsApproved", func() {
+			config := []byte("")
+			botConfig, err := parseConfigYAML(config)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(botConfig.HasLabelApproved()).To(BeFalse())
+			Expect(botConfig.LabelApproved()).To(BeNil())
+		})
+	})
+
+	When("the yaml file has label-approved only", func() {
+		It("returns no label-approved", func() {
+			config := []byte("label-approved:\n")
+			botConfig, err := parseConfigYAML(config)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(botConfig.HasLabelApproved()).To(BeFalse())
```

Comment:
Original intention was to allow not setting anything and the config to have defaults.

If we're changing that already, the simplest implementation would be to allow either no label-approved, or if it's specified then it much have the fields.

Reply:
I'm not changing this. It's the original behavior already, I'm just adding test.

For the yaml parser if you have:

```
label-approved:
```

It will set that field as nil